core/templates/dev/head/css/oppia.css

```diff
@@ -3111,3 +3112,275 @@ md-card.preview-conversation-skin-supplemental-card {
     float: right;
   }
 }
+
+.oppia-gallery-summary-tile {
```

Comment:
Sorry -- I got this wrong. I didn't realize the styles were being used for collections as well.

One request: can you rename this to oppia-activity-summary-tile? We have deprecated the term 'gallery', and these tiles are used in other places besides the library page anyway (like the profile page).